src/components/EggheadPlayer/use-egghead-player.ts

```diff
@@ -212,44 +213,69 @@ const onEnded = (lesson: {
   if (lesson.lesson_view_url) {
     onComplete(lesson)
   }
-
-  track('finished lesson', {
-    lesson: lesson.slug,
-    tags: lesson.tags.map((tag: {slug: any}) => tag.slug),
-  })
 }
 
 function onError() {}
 
-function onVideoQualityChanged() {}
-
-function onSubtitleChange() {}
-
-const getPlayerPrefs = () => cookies.get('egghead-player-prefs')
+const PLAY_PREFS_KEY = 'egghead-player-prefs'
+
+const defaultPlayerPreferences = {
+  volumeRate: 80,
+  playbackRate: 1,
+  autoplay: false,
+  videoQuality: {
+    bitrate: null,
+    height: null,
+    id: 'auto',
+    width: null,
+  },
+  subtitle: {
+    id: null,
+    kind: null,
+    label: 'off',
+    lang: null,
+  },
+  muted: false,
+}
 
-function onPlaybackRateChange(playbackRate: number) {
-  cookies.set('egghead-player-prefs', {...getPlayerPrefs(), playbackRate})
+export const getPlayerPrefs = () => {
```

Comment:
the `getPlayerPrefs` looks for the cookie and if it isn't there sets the cookie with our defaults 🏆